### DISCUSSION TITLE
Are bug reports for 1.12 still acceptable?

### DISCUSSION THREAD

**Discussion by pat2rome** [MrTJP/ProjectRed#1682] [Q&A] [answered]
I noticed that feedback buttons don't light up appropriately if they're already receiving a signal at world load until they also receive a block update, but my modpack is 1.12.2. It _is_ the latest version of the mod for that version of the game, but I understand if development has moved on entirely. I just wanted to check before I wasted my time and yours submitting an issue.

**Comment by pat2rome** [comment]
If I don't hear back in a week or so, I'll just post it. I'll be as thorough as I can with version details regardless.

**Comment by MrTJP** [comment, marked as answer]
Missed this question, since it wasn't filed under Q and A category. But to answer the question, no. If you have issues with 1.12, your solution is to try and update to the latest version.
